index.js

```diff
@@ -0,0 +1,380 @@
+						AES.decrypt(rows[0].password, `${process.env.AES_key}`)
+					) {
+						req.session.user = rows[0];
+						res.redirect("/dashboard");
+					} else {
+						console.log("Invalid password");
+						res.status(401).render("error", {
+							error_message: "Authentication error!"
+						});
+					}
+				} else {
+					console.log("Username not found!");
+					res.status(401).render("error", {
+						error_message: "Authentication error!"
+					});
+				}
+			} else {
+				console.log("Username not found!");
+				res.status(401).render("error", {
+					error_message: "Authentication error!"
+				});
+			}
+		}
+	);
+});
+app.post("/checkAvailability", (req, res) => {
+	if (req.session.user) {
+		con.query(
+			"SELECT slot_hall FROM booking WHERE event_id IN (SELECT id FROM events WHERE date_wanted = ?)",
+			req.body.date_wanted,
+			(err, result) => {
+				if (!err) {
+					res.send(result);
+				} else {
+					console.log(err);
+					res.status(500).render("error", {
+						error_message: "Availability checking failed!"
+					});
+				}
+			}
+		);
+	} else {
+		res.redirect("/login");
+	}
+});
+app.post("/makeRequest", (req, res) => {
+	if (req.session.user) {
+		var user_id = req.session.user.id;
+		var date_wanted = req.body.date_wanted;
+		var slot_hall = "";
+		for (var i = 1; i <= 7; i++)
+			for (var j = 1; j <= 9; j++) {
+				if (req.body[`cell${j}${i}`]) {
+					slot_hall += j.toString() + i.toString();
+				}
+			}
+		var club = req.body.club_name;
+		var details = req.body.desc;
+		var title = req.body.event_name;
+		var requirements = req.body.requirements;
+		var event_obj = {
+			title: title,
+			club: club,
+			requirements: requirements,
+			details: details,
+			date_wanted: date_wanted,
+			user_id: user_id
+		};
+		con.query("insert into events set ?", event_obj, err => {
+			if (!err) {
+				console.log(`Inserted ${event_obj.title} into events.`);
+				con.query(
+					"select id from events WHERE user_id= ? ORDER BY id DESC LIMIT 1",
+					req.session.user.id,
+					(err, event_id) => {
+						if (!err) {
+							var booking_obj = {
+								user_id: user_id,
+								event_id: event_id[0].id,
+								slot_hall: slot_hall
+							};
+							con.query(
+								"insert into booking set ?",
+								booking_obj,
+								err => {
+									if (!err) {
+										console.log(
+											`Inserted event with id ${booking_obj.event_id} into booking.`
+										);
+										res.redirect("/dashboard");
+									} else {
+										console.log(err);
+										res.status(500).render("error", {
+											error_message:
+												"Inserting into booking failed!"
+										});
+									}
+								}
+							);
+						} else {
+							console.log(err);
+							res.status(500).render("error", {
+								error_message: "Retrieving event_id failed!"
+							});
+						}
+					}
+				);
+			} else {
+				console.log(err);
+				res.status(500).render("error", {
+					error_message: "Inserting into event failed!"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'error_message' is not in camel case. (camelcase)](https://app.codacy.com/app/AravindVNair99/Hall-Management-System/pullRequest?prid=4592231)